### PR TITLE
GH-310: Updated Dependencies

### PR DESCRIPTION
**Pull request by mattleibow**
### Description of Change ###

Updating the NuGet versions and cleaned the project files.

### Bugs Fixed ###

- Related to issue #310 

### API Changes ###

None.

### Behavioral Changes ###

None.

### PR Checklist ###

- [x] Has tests
- [x] Has samples 
- [x] Rebased on top of master at time of PR
- [x] Changes adhere to coding standard
- [x] Updated documentation

### CODE REVIEW

**Comment by dend** [comment]
### :x: Validation status: errors


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[]() | :x:Error | | [Details](#user-content-)

<a name=""></a>
### []()
 - **[Error]** Cannot sync git repo to specified state. Please make sure the pull request is mergeable. Details: Git merge results in conflict, repository remote url: https://github.com/xamarin/Essentials, working directory: W:\yxxb-s, target branch: master, target branch commit: b05a2667c4af2e7b0f2977041ac4a4d9720972a5, source branch: dev/update-deps, source branch commit: .

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C6%5C25%5Cf63aadd0-c8f5-13ca-04e7-037e45f8c0b9%5CPullRequest%5C201806251559457193-324%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=SGU5EFFBc3%2FTPKx2Elktv02hpxeS%2FXkzy%2B5L%2FGdn7IU%3D&st=2018-06-25T15%3A54%3A53Z&se=2018-07-26T15%3A59%3A53Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by dend** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[DeviceTests/DeviceTests.Android/DeviceTests.Android.csproj](https://github.com/xamarin/Essentials/blob/dev/update-deps/DeviceTests/DeviceTests.Android/DeviceTests.Android.csproj) | :white_check_mark:Succeeded | |
[DeviceTests/DeviceTests.UWP/DeviceTests.UWP.csproj](https://github.com/xamarin/Essentials/blob/dev/update-deps/DeviceTests/DeviceTests.UWP/DeviceTests.UWP.csproj) | :white_check_mark:Succeeded | |
[DeviceTests/DeviceTests.iOS/DeviceTests.iOS.csproj](https://github.com/xamarin/Essentials/blob/dev/update-deps/DeviceTests/DeviceTests.iOS/DeviceTests.iOS.csproj) | :white_check_mark:Succeeded | |
[Samples/Samples.Android/Samples.Android.csproj](https://github.com/xamarin/Essentials/blob/dev/update-deps/Samples/Samples.Android/Samples.Android.csproj) | :white_check_mark:Succeeded | |
[Samples/Samples.UWP/Samples.UWP.csproj](https://github.com/xamarin/Essentials/blob/dev/update-deps/Samples/Samples.UWP/Samples.UWP.csproj) | :white_check_mark:Succeeded | |
[Samples/Samples.iOS/Samples.iOS.csproj](https://github.com/xamarin/Essentials/blob/dev/update-deps/Samples/Samples.iOS/Samples.iOS.csproj) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/Xamarin.Essentials.csproj](https://github.com/xamarin/Essentials/blob/dev/update-deps/Xamarin.Essentials/Xamarin.Essentials.csproj) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C6%5C25%5Cf63aadd0-c8f5-13ca-04e7-037e45f8c0b9%5CPullRequest%5C201806251602178120-324%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=deRErUWiNyBMsVNL%2BeK10564EJiwy9PbRJ94NH7Bgw4%3D&st=2018-06-25T15%3A58%3A12Z&se=2018-07-26T16%3A03%3A12Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by mattleibow** [comment]
This was an update PR, but I forgot to actually create it. This PR now just cleans up the .csproj, removing the multiple `<ItemGroup>`s and orders/groups the items correctly

File: Xamarin.Essentials/Xamarin.Essentials.csproj [hunk 1]
```diff
@@ -51,7 +51,7 @@
     <Compile Include="**\*.netstandard.*.cs" />
   </ItemGroup>
   <ItemGroup Condition=" $(TargetFramework.StartsWith('uap10.0')) ">
-    <PackageReference Include="Microsoft.NETCore.UniversalWindowsPlatform" Version="6.0.6" />
+    <PackageReference Include="Microsoft.NETCore.UniversalWindowsPlatform" Version="6.1.5" />
```
Comment (jamesmontemagno) [inline Xamarin.Essentials/Xamarin.Essentials.csproj]: do we want to bump to this? I assume it is okay? will it align with templates and does it require specific versions of .NET Core to be installed?

Comment (mattleibow) [reply Xamarin.Essentials/Xamarin.Essentials.csproj]: This is not related to .net core (MS just used that name for some reason). This is the new version in the templates: https://github.com/xamarin/xamarin-templates/search?q=UniversalWindowsPlatform&unscoped_q=UniversalWindowsPlatform

Comment (jamesmontemagno) [reply Xamarin.Essentials/Xamarin.Essentials.csproj]: Should just sync with @pierceboggan to make sure our templates use this version too.

**Comment by dend** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[DeviceTests/DeviceTests.Android/DeviceTests.Android.csproj](https://github.com/xamarin/Essentials/blob/dev/update-deps/DeviceTests/DeviceTests.Android/DeviceTests.Android.csproj) | :white_check_mark:Succeeded | |
[DeviceTests/DeviceTests.UWP/DeviceTests.UWP.csproj](https://github.com/xamarin/Essentials/blob/dev/update-deps/DeviceTests/DeviceTests.UWP/DeviceTests.UWP.csproj) | :white_check_mark:Succeeded | |
[DeviceTests/DeviceTests.iOS/DeviceTests.iOS.csproj](https://github.com/xamarin/Essentials/blob/dev/update-deps/DeviceTests/DeviceTests.iOS/DeviceTests.iOS.csproj) | :white_check_mark:Succeeded | |
[Samples/Samples.Android/Samples.Android.csproj](https://github.com/xamarin/Essentials/blob/dev/update-deps/Samples/Samples.Android/Samples.Android.csproj) | :white_check_mark:Succeeded | |
[Samples/Samples.UWP/Samples.UWP.csproj](https://github.com/xamarin/Essentials/blob/dev/update-deps/Samples/Samples.UWP/Samples.UWP.csproj) | :white_check_mark:Succeeded | |
[Samples/Samples.iOS/Samples.iOS.csproj](https://github.com/xamarin/Essentials/blob/dev/update-deps/Samples/Samples.iOS/Samples.iOS.csproj) | :white_check_mark:Succeeded | |
[Xamarin.Essentials/Xamarin.Essentials.csproj](https://github.com/xamarin/Essentials/blob/dev/update-deps/Xamarin.Essentials/Xamarin.Essentials.csproj) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C6%5C25%5Cf63aadd0-c8f5-13ca-04e7-037e45f8c0b9%5CPullRequest%5C201806251806177630-324%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=Mai0kX%2FhOvm015p1lEFAZ8XaSktFK1hqIIPjf3sApNo%3D&st=2018-06-25T18%3A02%3A15Z&se=2018-07-26T18%3A07%3A15Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**